src/common/components/breadcrumbs/Breadcrumbs.tsx

```diff
@@ -1,35 +1,36 @@
 import React from 'react';
 import { ChevronRightIcon, HomeIcon } from '@heroicons/react/solid';
 import { NavLink, useLocation } from 'react-router-dom';
-import useBreadcrumbs from "use-react-router-breadcrumbs";
-import { useSelectedOrganization, useSelectedPracticeProgram } from '../../../store/Selectors';
+import useBreadcrumbs from 'use-react-router-breadcrumbs';
 import i18n from '../../configs/i18n';
 import { MENU_ROUTES_HREF } from '../../constants/Menu.constants';
 
 const DynamicPracticeProgram = ({ match }: any) => {
-  const { selectedProgram } = useSelectedPracticeProgram();
+  // const { selectedProgram } = useSelectedPracticeProgram();
```

Comment:
Lot of comments

Reply:
https://wearetribus.atlassian.net/browse/C4R-1514